lib/bundlex/cnode/server.ex

```diff
@@ -84,6 +84,12 @@ defmodule Bundlex.CNode.Server do
     {:stop, reason, state}
   end
 
+  def handle_info({:cnode, packet}, state) do
+    # forwards packet to caller
```

Comment:
Such comments don't add any value, while they need to be maintained, so please avoid them.